test/test_run_db_test_docker_db_check_arguments.py

```diff
@@ -0,0 +1,87 @@
+import unittest
+from pathlib import Path
+
+import docker
+import os.path
+from exasol_integration_test_docker_environment.test import utils
+from exasol_integration_test_docker_environment.lib.data.environment_info import EnvironmentInfo
+
+import utils as exaslct_utils
+
+
+class DockerRunDBTestDockerDBTestCheckArguments(unittest.TestCase):
+
+    def setUp(self):
+        print(f"SetUp {self.__class__.__name__}")
+        self.test_environment = utils.ExaslctTestEnvironment(self, exaslct_utils.EXASLCT_DEFAULT_BIN)
+        self.test_environment.clean_images()
+        self.client = docker.from_env()
+
+    def tearDown(self):
+        self.remove_docker_runtime()
+        self.test_environment.close()
+        self.client.close()
+
+    def _getEnvironmentInfo(self):
+        test_environment_name = f"""{self.test_environment.flavor_path.name}_release"""
+        env_path = os.path.join(self.test_environment.temp_dir, "cache", "environments", test_environment_name, "")
+        environment_info_json_path = Path(self.test_environment.temp_dir,
+                                          f"cache/environments/{test_environment_name}/environment_info.json")
+        if environment_info_json_path.exists():
+            with environment_info_json_path.open() as f:
+                return EnvironmentInfo.from_json(f.read())
+
+    def assert_mem_disk_size(self, mem_size: str, disk_size: str):
+        env_info = self._getEnvironmentInfo()
+        containers = \
+            [c.name for c in
+             self.client.containers.list()
+             if env_info.database_info.container_info.container_name in c.name]
+        db_container = [c for c in containers if "db_container" in c]
+        exit_result = self.client.containers.get(db_container[0]).exec_run("cat /exa/etc/EXAConf")
+        output = exit_result[1].decode("UTF-8")
+        if output == '':
+            exit_result = self.client.containers.get(db_container[0]).exec_run("cat /exa/etc/EXAConf")
+            output = exit_result[1].decode("UTF-8")
+            return_code = exit_result[0]
+        return_code = exit_result[0]
+        self.assertEqual(return_code, 0)
+        self.assertIn(f"MemSize = {mem_size}", output)
+        self.assertIn(f" Size = {disk_size}", output)
+
+    def remove_docker_runtime(self):
```

Comment:
```suggestion
    def remove_docker_environment(self):
```

docker runtime is misleading